src/gui/ckbsystemtrayicon.cpp

```diff
@@ -25,7 +25,6 @@ void CkbSystemTrayIcon::setIcon(QIcon icon, QString name)
     }
 
     // If not, load the internal ones
-    qDebug() << "Indicator icon not found in theme hicolor";
```

Comment:
Why was this removed?

Reply:
Because it spammed the logs every time the icon updated. I'll add it again, since it should no longer print anything when the icon is installed into the theme.

Reply:
Ah, that makes sense. That thought didn't even cross my mind for some reason.

I did put it there so that it's easy to tell the fallback code is being used, as it forces a 22x22 resize, which will look pretty terrible on high dpi screens.
If after being readded it ends up being an issue, I'll find a way to make it only complain once, but for now, as you mentioned, it shouldn't be a problem.
Thanks!